static-assets/components/cstudio-dialogs/upload-flickr-dialog.js

```diff
@@ -235,51 +235,6 @@ CStudioAuthoring.Dialogs.UploadFlickrDialog = CStudioAuthoring.Dialogs.UploadFli
     YAHOO.util.Connect.asyncRequest('POST', serviceUri, uploadHandler);
   },
 
-  /**
-   *
-   */
-  overwritePopupSubmit: function(event, args) {
```

Comment:
Deleted?

Reply:
that's an unused piece of code (in several dialogs)